src/components/CreateEvent/CreateEventNavbar.jsx

```diff
@@ -39,40 +34,45 @@ const CreateEventNavbar = () => {
         </div>
       </div>
 
-      <div className="">
-        {open && (
-          <ul className={menuStyles.theCreateEventContainerLinks}>
-            <Link to="/dashboard/upcoming_events" className={menuStyles.goTo}>
-              Dashboard
-            </Link>
-            <Link to="/profile" className={menuStyles.goTo}>
-              Account Setting
-            </Link>
-            <Link to="/help/*" className={menuStyles.goTo}>
-              Help and Support
-            </Link>
-            <Link to="/error404" className={menuStyles.goTo}>
-              Notification
-            </Link>
-            <Link to="/error404" className={menuStyles.syncWithGoogle}>
-              <img src={googleCalendar} alt="" />
-              Sync with Google Calender
-            </Link>
-
-            <span className={menuStyles.theUserDetail}>
-              <div className="w-[35px] h-[35px] border border-[#0056D6] text-[#717172] rounded-full flex justify-center items-center object-cover text-xs">
-                <span>M</span>
-              </div>
-              <span className={menuStyles.theUsersName}>
-                <span>Mbulu Benita</span>
-                <span>beniottabenita@gmail.com</span>
-              </span>
-            </span>
-            <Link to="/error404" className={menuStyles.logOutBtn}>
-              <span>Log out</span>
-            </Link>
-          </ul>
-        )}
+      <div class="text-left">
+        {open && 
+          <div class="absolute right-6 z-10 w-56 origin-top-right divide-y divide-gray-100 rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none" role="menu" aria-orientation="vertical" aria-labelledby="menu-button" tabindex="-1">
+            <div class={`${menuStyles.goTo} py-3 px-2`} role="none">
+              <ul>
+                <li>
+                  <BsPerson />
+                  <Link to="/dashboard/upcoming_events" className="block px-2 py-2 text-sm">
+                    Profile
+                  </Link>
```

Comment:
- [ ] Kindly use the correct link for the profile

Here is the link 👇🏽 
`<Route path="/profile" element={<ProfilePage />} />`

Reply:
Okay